packages/web-components/demo/install-ses-lockdown.js

```diff
@@ -2,7 +2,7 @@
 import '@endo/init/pre-remoting.js';
 
 lockdown({
-  errorTaming: 'unsafe',
+  errorTaming: 'unsafe', // TODO Wen safe?
```

Comment:
This is just meant to be run locally, I think it's always "dev" mode, just a demo page so unsafe is probably okay.

Reply:
I renamed to make the implied unsafety of this file clearer. Resolving.